page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/tasks/TaskHandler.java

```diff
@@ -5,18 +5,19 @@
 
 public class TaskHandler {
 
-    private static List<Task> tasks = new ArrayList<>();
+    private static ThreadLocal<List<Task>> tasks = ThreadLocal.withInitial(() -> new ArrayList<Task>());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. ([squid:S1612](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1612))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOEZcrQ7oSxgxnf_&open=AWeIHOEZcrQ7oSxgxnf_)

[ISSUE_KEY]: AWeIHOEZcrQ7oSxgxnf_